### PR TITLE
Add chapter 7 bibliography and link citations. Closes #125

### LINKED ISSUE #125: Chapter 7 Citation To-Dos
[Link](https://bookdown.org/yihui/bookdown/citations.html) to Bookdown citations documentation.
- [x] Read through chapter text, making sure that each citation is present and properly formatted.
- [x] Add citations from book.bib file to a new collection for the chapter in Zotero.
- [x] Add any missing citations manually into Zotero.
- [x] Create new branch for the chapter.
- [x] Export the chapter's .bib file as BibTeX and save into the bibliographies folder within textbook.
- [x] Add the .bib file into the array labeled "bibliography" within the index.Rmd file.
- [x] Link each inline citation to its .bib listing equivalent.
- [x] Make sure that each inline citation links to the corresponding reference within the bibliography section of the chapter.

### PR DESCRIPTION
**Pull request by nickmmurphy**

### CODE REVIEW

File: 07-topology-and-geocoding.Rmd [hunk 1]
```diff
@@ -485,13 +480,13 @@ In this next section, a number of topologies are described that are important fo
...
 
 ``` {r 7-platonic-polyhedrons, echo=FALSE, out.width = '75%', fig.cap=fig_cap}
-fig_cap <- paste0("The five Platonic solids are examples of regular, convex polyhedrons and multipatch geometries. From left to right: tetrahedron (4 faces); hexahedron (6 faces); octahedron (8 faces); dodecahedron (12 faces); and icosahedron (20 faces). <a href='",baseurl,"#7-platonic-polyhedrons'>Animated figure can be viewed in the web browser version of the textbook.</a> Kjell André, CC-BY-SA-3.0.")
+fig_cap <- paste0("The five Platonic solids are examples of regular, convex polyhedrons and multipatch geometries. From left to right: tetrahedron (4 faces); hexahedron (6 faces); octahedron (8 faces); dodecahedron (12 faces); and icosahedron (20 faces). <a href='",baseurl,"#7-platonic-polyhedrons'>Animated figure can be viewed in the web browser version of the textbook.</a> @cyp_polyhedra_2005, CC-BY-SA-3.0.")
```
Comment (ect123) [inline 07-topology-and-geocoding.Rmd:483]: does `@cyp_polyhedra_2005` need brackets?

File: 07-topology-and-geocoding.Rmd [hunk 2]
```diff
@@ -537,7 +532,7 @@ We can derive some useful information from these 3D convex alpha hulls. One obvi
 Similar to Thiessen polygons, Voronoi tessellations can be undertaken for 3D point clouds. 3D Voronoi tessellations produce nearest neighbour polyhedrons around the space of each 3D point. Just as in the 2D case, these 3D Voronoi tessellations can be used to partition the 3D space. Figure \@ref(fig:7-neper) shows the use of a 3D Voronoi tessellation for use in mapping and visualizing polycrystals.
 
 ```{r 7-neper, echo=FALSE, out.width = '75%', fig.cap=fig_cap}
-fig_cap <- paste("3D Voronoi tesselation of simulated polycrystals. <a href='https://github.com/rquey/neper' target='_blank'>Neper</a>, GPL 3.0")
+fig_cap <- paste("3D Voronoi tesselation of simulated polycrystals. @quey_neper_nodate, GPL 3.0.")
```
Comment (ect123) [inline 07-topology-and-geocoding.Rmd:535]: Also, brackets?

**Review by ect123 — CHANGES_REQUESTED** [review]
This all looks great, but just want to confirm that the two flagged citations don't need brackets.

File: 07-topology-and-geocoding.Rmd (hunk 1 above)
Comment (nickmmurphy) [reply 07-topology-and-geocoding.Rmd:483]: @ect123 So the way I've been adding the citations to figures is by following how Paul has been putting his name and license info, for ex "Caption text here. Pickell, CC-BY-SA." So the way this displays here is "Caption text here. Cyp (2005), CC-BY_SA." It was intentional but is an easy fix if needed!

Comment (ect123) [reply 07-topology-and-geocoding.Rmd:483]: Great! should be good to go - just wanted to make double sure the syntax wasn't going to throw anything off.

**Pull request merged**